src/main/python/apache/thermos/runner/thermos_runner.py

```diff
@@ -267,7 +267,7 @@ def main(args, opts):
 
 
 LogOptions.set_simple(True)
-LogOptions.set_disk_log_level('DEBUG')
+LogOptions.set_disk_log_level('INFO')
```

Comment:
For debugging issues in production, I sometimes really needed those DEBUG logs. I think a saner option would be to remove that line here entirely and instead pass over the log level of the executor. This would provide us with always the same log level in executor and runner, but as a tradeoff this is fair, I think.

We currently already pass over the log level so this should be easy to extend https://github.com/apache/aurora/blob/32776792d273b36afbf4a1bab69a66fb06163ffd/src/main/python/apache/aurora/executor/thermos_task_runner.py#L241